src/main/java/org/broadinstitute/hellbender/tools/spark/CreateHadoopBamSplittingIndex.java

```diff
@@ -32,6 +34,7 @@
 @CommandLineProgramProperties(summary = "Create a HadoopBam splitting index as well as a bam index from a bam file.",
         oneLineSummary = "create a hadoop-bam splitting index" ,
         programGroup = SparkProgramGroup.class)
+@DocumentedFeature(groupName= HelpConstants.DOCS_CAT_DATA)
```

Comment:
@vdauwera @lbergelson Ok then, I think we can pretty easily achieve what we want by just consolidating ALL of the group names and descriptions (including the ones that are currently embedded in CommandLineProgramGroup derived classes) into HelpConstants, and then just writing a method there that maps a program group string to a "sparky" property and a "supercat" property. Then all the mapping code will be in one place, and we can remove the gross code in the custom doclet that currently parses the names to get a supercat, and replace it with a call to the new method. That, in combination with a change to Barclay to fallback to get the group name from CommandLineProgramProperties (I did only did this for summary but should do it for group name as well), I think will get us what we want.

If we agree on that, I'll make those changes.